---
titre: "Novel Insights into Quantitative Proteomics from an Innovative Bottom-Up Simple Light Isotope Metabolic (bSLIM) Labeling Data Processing Strategy"
auteurs: "<b>Nicolas Sénécaut</b>, Gelio Alves, Hendrik Weisser, Laurent Lignières, Samuel Terrier, Lilian Yang-Crosson, Pierre Poulain, Gaëlle Lelandais, Yi-Kuo Yu, and Jean-Michel Camadro"
DOI: "10.1021/acs.jproteome.0c00478"
journal: "Journal of Proteome Research"
PMID: "" 
PMCID: "" 
year: 2021
PDF: "bSLIM.pdf"
---

### Abstract

**Title**: Novel Insights into Quantitative Proteomics from an Innovative Bottom-Up Simple Light Isotope Metabolic (bSLIM) Labeling Data Processing Strategy

**Abstract**: Simple light isotope metabolic labeling (SLIM labeling) is an innovative method to quantify variations in the proteome
based on an original *in vivo* labeling strategy. Heterotrophic cells grown in U-[<sup>12</sup>C] as the sole source of carbon synthesize U-[<sup>12</sup>C]-
amino acids, which are incorporated into proteins, giving rise to U-[<sup>12</sup>C]-proteins. This results in a large increase in the intensity of the
monoisotope ion of peptides and proteins, thus allowing higher identification scores and protein sequence coverage in mass spectrometry
experiments. This method, initially developed for signal processing and quantification of the incorporation rate of 12C into peptides,
was based on a multistep process that was difficult to implement for many laboratories. To overcome these limitations, we developed
a new theoretical background to analyze bottom-up proteomics data using SLIM-labeling (bSLIM) and established simple procedures
based on open source software, using dedicated OpenMS modules, and embedded R scripts to process the bSLIM experimental data.
These new tools allow both computation of the <sup>12</sup>C abundance in peptides to follow the kinetics of protein labeling and the molar
fraction of unlabeled and <sup>12</sup>C-labeled peptides in multiplexing experiments to determine the relative abundance of proteins extracted
under different biological conditions. They also make it possible to consider incomplete <sup>12</sup>C labeling, such as that observed in cells with
nutritional requirements for non-labeled amino acids. These tools were validated on an experimental dataset produced using various
yeast strains of *Saccharomyces cerevisiae* and growth conditions. The workflows are built on the implementation of appropriate calculation
modules in a KNIME working environment. These new integrated tools provide a convenient framework for the wider use of the
SLIM-labeling strategy.

**Keywords**: *In vivo* metabolic labeling, light carbon isotope, <sup>12</sup>C, quantitative proteomics, data processing workflow, OpenMS, KNIME, Yeast